tests/Unit/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -241,9 +162,9 @@ void vector_test_construct_and_assign(
 template <typename VectorType, typename ValType>
 void vector_test_serialize(
     typename tt::get_fundamental_type_t<ValType> low =
-        typename tt::get_fundamental_type_t<ValType>{-100.0},
+        typename tt::get_fundamental_type_t<ValType>{-100},
     typename tt::get_fundamental_type_t<ValType> high =
-        typename tt::get_fundamental_type_t<ValType>{100.0}) noexcept {
+        typename tt::get_fundamental_type_t<ValType>{100}) noexcept {
```

Comment:
No need for `typename` here and why not just `const tt::get_fundamental_type_t low = static_cast<tt::get_fundamential_type_t>(100)`?